packages/bridge-ui-v2/src/components/Header/Header.svelte

```diff
@@ -27,13 +25,11 @@
     md:px-10
     md:py-7
  ">
-  <LogoWithText width={150} height={50} class="md:hidden" />
+  <LogoWithText width={77} height={21} class="md:hidden" />
 
   <div class="flex justify-end md:f-between-center w-full">
     {#if isBridgePage}
       <BridgeTabs class="hidden md:flex" />
-    {:else}
-      <div></div>
```

Comment:
This is here on purpose, otherwise it looks like this on every other page:
![image](https://github.com/taikoxyz/taiko-mono/assets/5267230/a4c22904-1c50-4a2b-950c-0b788167e001)

Reply:
No no, adding an empty div to make something work is a hack, we can do this with CSS, no empty divs. I missed other pages. I'll fix it

Reply:
Fixed. We just need to justify-end for all the children, and the tabs would take up the rest of the space. By default the alignment inside is left, so it'll go all the way to the left